float32[6] ControllerInput
